File: data/gcc_versions_comparison/src/program/src/func44.c
#include "program.h" 
array_t* func44(array_t_param* vars, int loopsFactor) {
   size_t pCounter = vars->size;
   array_t* array21;
   if (pCounter > 0) {
      array21 = vars->data[--pCounter];
      array21->refC++;
      DEBUG_COPY(array21->id);
   } else {
      array21 = (array_t*)malloc(sizeof(array_t));
      array21->size = 87;
      array21->refC = 1;
      array21->id = 21;
      array21->data = (unsigned int*)malloc(array21->size*sizeof(unsigned int));
      memset(array21->data, 0, array21->size*sizeof(unsigned int));
      DEBUG_NEW(array21->id);
   }
   return array21;
}

File: data/multilang_comparison/program_julia/src/func44.jl
include("program_julia_head.jl")
function func44(vars::JArray_param, loopsFactor::Int)::JArray
   pCounter = length(vars.data)
   array21 = nothing
   if pCounter > 1
       array21 = vars.data[pCounter]
       pCounter -= 1
       array21.refC += 1
   else
       array21 = JArray(zeros(Int, 87), 87, 1, 21)
   end
   return array21;
end

